ui/js/graphs.js

```diff
@@ -299,12 +299,14 @@ perf.controller('GraphsCtrl', [
     function plotGraph() {
       // synchronize series visibility with flot, in case it's changed
       $scope.seriesList.forEach(function(series) {
-        series.flotSeries.points.show = series.visible;
+          series.flotSeries.points.show = series.visible;
+          series.active = (!series.subtestSignatures || $scope.myMeasure === 'mean');
+          series.blockColor = series.active ? series.color : "grey";
       });
 
       // reset highlights
       $scope.seriesList.forEach(function(series) {
-        series.highlightedPoints = [];
+          series.highlightedPoints = [];
```

Comment:
Remove the indentation change (i.e. the two extra spaces). We should move to 4 space indentation at some point to be consistent with the rest of perfherder but not as part of this.